proposals/features-legacyIds-deprecated-properties.md

```diff
@@ -0,0 +1,66 @@
+Goal: Adds support for renaming and deprecating a Feature.
+
+Adding two new properties to the [devcontainer-feature.json](../docs/specs/devcontainer-features.md#devcontainer-featurejson-properties) 👇 
+
+# 1. legacyIds
+
+| Property | Type | Description |
+| :--- | :--- | :--- |
+| `legacyIds` | array | Defaults to `[]`. Array of old IDs used to publish this Feature. The property is useful for renaming a currently published Feature within a single namespace. |
+
+- The [source code](../docs/specs/features-distribution/#source-code) can be changed to a new name along with [devcontainer-feature.json properties](../docs/specs/devcontainer-features.md#devcontainer-featurejson-properties) (like `id`, `name`, `documentationURL`).
```

Comment:
I know this is just the proposal, but when added into the spec, it would be cool to have a step-by-step guide on how to do this, like so:

```suggestion
1. Update the Feature [source code](../docs/specs/features-distribution/#source-code) folder and [devcontainer-feature.json properties](../docs/specs/devcontainer-features.md#devcontainer-featurejson-properties) to reflect the updated `id`. Other properties (`name`, `documentationUrl`, etc.) can optionally be updated during this step.
2. Add or update the `legacyIds` property to the Feature, including the previously used `id`.
3. Rerun the `devcontainer features publish` command, or equivalent tool that implements the [Features distribution specification](../docs/specs/features-distribution/#distribution)  
```